extension/src/terminal/GradleRunnerTerminal.ts

```diff
@@ -108,7 +108,7 @@ export class GradleRunnerTerminal implements vscode.Pseudoterminal {
     } catch (e) {
       this.handleError(e);
```

Comment:
In normal workflow,  `this.closeEmitter.fire(0);`.
In catch hander, `this.closeEmitter.fire(1);`